python_modules/libraries/dagster-dbt/dagster_dbt/__init__.py

```diff
@@ -141,6 +144,33 @@
             ("dbt_test_op", "dagster_dbt.ops"),
         ]
     },
+    **{
+        value: (
+            module,
+            "0.24.0",
+            f"Use the @dbt_assets decorator, DbtCliResource, and DagsterDbtTranslator instead. {additional_text}",
+        )
+        for value, module, additional_text in [
+            (
+                "load_assets_from_dbt_manifest",
+                "dagster_dbt.asset_defs",
+                (
+                    "To generate a dbt manifest at run time in your Python code using `dbt parse`,"
+                    " see the reference:"
+                    " https://docs.dagster.io/integrations/dbt/reference#loading-dbt-models-from-a-dbt-project.",
+                ),
+            ),
+            (
+                "load_assets_from_dbt_project",
+                "dagster_dbt.asset_defs",
+                (
+                    "To generate a dbt manifest at build time in your CI/CD system using"
```

Comment:
It's a little unclear to me how this is related to the deprecation - wouldn't they already need to be doing this if they're using  `load_assets_from_dbt_manifest`?

I think the main question I would have for migrating from `load_assets_from_dbt_manifest` is what I would need to put in the body of the `@dbt_assets`-decorated function in order to get identical behavior.

Reply:
Ok, I've removed this snippet. I'll only include a snippet for generating the manifest at run time for `load_assets_from_dbt_project`.

For both functions, I've added a link to the API docs so that they have examples on how to execute `dbt run` or `dbt build` when using these new functions (since these were the only two functions available for `load_assets_from_dbt_XXX`). I've also added a link on customizing the asset metadata using `DagsterDbtTranslator`.

Once we have a step-by-step guide for the migration, we can also link it here.